microsoft-edge/dualengine/reference/accelerator.md

```diff
@@ -0,0 +1,55 @@
+---
+description: Contains the information representing a keyboard accelerator.
+title: DualEngine Win32 C++ ACCELERATOR
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.prod: microsoft-edge
+ms.date: 12/06/2023
+keywords: dual engine, dualengine, iemode, win32 apps, win32, edge, ie mode, edge html, ACCELERATOR
+topic_type: 
+- APIRef
+api_name:
+- ACCELERATOR
+- ACCELERATOR.eventType
+- ACCELERATOR.flags
+- ACCELERATOR.keyCode
+api_type:
+- COM
+api_location:
+- ieframe.dll
+---
+
+# struct ACCELERATOR
+
+> [!IMPORTANT]
+> The DualEngine API is a limited access feature. Contact dualengineapiaccess@microsoft.com for more information.
+
+Contains the information representing a keyboard accelerator (keyboard shortcut).

```

Comment:
```suggestion
Represents a keyboard accelerator (keyboard shortcut).
```
concise version

Reply:
@AndrewLakeMSFT - example of a change i made/pushed commit, in a generated (transient) Ref .md file.  You can see that change in latest Files Changed tab, & rendered topic.  I can Resolve this comment, but first, did you make this particular change in the actual source file?